job-service-acceptance-tests/src/test/java/com/hpe/caf/jobservice/acceptance/JobServiceDatabaseUtil.java

```diff
@@ -95,8 +96,8 @@ public static void assertJobRowExists(final String jobId) throws SQLException
             st.setString(1, jobId);
             final ResultSet jobRS = st.executeQuery();
             jobRS.next();
-            Assert.assertEquals(jobRS.getString(1), jobId);
-            Assert.assertTrue(!jobRS.next());
+            Assert.assertEquals(jobRS.getString(1), jobId, "Job "+jobId+" exists in job table");
+            Assert.assertTrue(!jobRS.next(), "No dupliacte job "+jobId+" exists in job table");
```

Comment:
This is asserting that the job does not exist, if the assert fails then the message is displayed so the message should say that the duplicate job does exist.